File: APIRollTheDice-Backend/src/main/java/com/example/APIRollTheDice/Model/DTO/GameDTO/Storage/StorageItemDTO.java
package com.example.APIRollTheDice.Model.DTO.GameDTO.Storage;

public class StorageItemDTO {
    private Long id;
    private Long idCustomObject;
    private int quantity;
    private Long idStorage;

    public StorageItemDTO() {
    }

    public StorageItemDTO(Long id, Long idCustomObject, int quantity, Long idStorage) {
        this.id = id;
        this.idCustomObject = idCustomObject;
        this.quantity = quantity;
        this.idStorage = idStorage;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getIdCustomObject() {
        return idCustomObject;
    }

    public void setIdCustomObject(Long idCustomObject) {
        this.idCustomObject = idCustomObject;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public Long getIdStorage() {
        return idStorage;
    }

    public void setIdStorage(Long idStorage) {
        this.idStorage = idStorage;
    }
}

File: RollTheDice/Assets/_Project/API/Model/DTO/GameDTO/StorageDTO/StorageItemDTO.cs
namespace Assets._Project.API.Model.DTO.GameDTO.StorageDTO
{
        public class StorageItemDTO{
        public long Id { get; set;}
        public long IdCustomObject { get; set;}
        public int Quantity { get; set;}
        public long IdStorage { get; set;}

        public StorageItemDTO(){}
        public StorageItemDTO(long id, long idCustomObject, int quantity, long idStorage)
        {
            Id = id;
            IdCustomObject = idCustomObject;
            Quantity = quantity;
            IdStorage = idStorage;
        }
    }
}